### ISSUE TITLE
Proposal: Dev command to empty the database

### ISSUE THREAD

**Issue by Kukant** [MarquezProject/marquez#2489]
Currently, when I want to empty the db, I need to use docker to recreate the database volume. I would like a command, script or API endpoint that would allow me to do this. 

I know about soft delete, but that is too soft for me.

**Comment by wslulciuc** [comment]
@Kukant, I agree. With PR https://github.com/MarquezProject/marquez/pull/2486, you will be able to drop all metadata for one or more namespaces. We will begin to expose more admin specific tasks like:

 * `/tasks/db-clear`
 * `/tasks/db-retention`
 * and more!
